daffodil-cli/src/conf/.keep

```diff
@@ -0,0 +1,14 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I recall it being a bit of a pain to find the documentation in sbt-native-packager about how to add config files, so I want to keep as much of it around as possible in case we ever need to add some other config file. For example, there are still variables in the CLI sh/bat scripts to locate this directory. For now it's just an empty directory and nothing actually uses it.